q2_phylogeny/_fasttree.py

```diff
@@ -21,14 +22,22 @@ def run_command(cmd, output_fp, verbose=True):
               "no longer exist.")
         print("\nCommand:", end=' ')
         print(" ".join(cmd), end='\n\n')
+
+    if env is None:
+        env = os.environ.copy()
     with open(output_fp, 'w') as output_f:
-        subprocess.run(cmd, stdout=output_f, check=True)
+        subprocess.run(cmd, stdout=output_f, check=True, env=env)
 
 
-def fasttree(alignment: AlignedDNAFASTAFormat) -> NewickFormat:
+def fasttree(alignment: AlignedDNAFASTAFormat,
+             n_threads: int=1) -> NewickFormat:
     result = NewickFormat()
     aligned_fp = str(alignment)
     tree_fp = str(result)
-    cmd = ['FastTree', '-quote', '-nt', aligned_fp]
-    run_command(cmd, tree_fp)
+
+    env = os.environ.copy()
+    env.update({'OMP_NUM_THREADS': str(n_threads)})
+
+    cmd = ['FastTreeMP', '-quote', '-nt', aligned_fp]
```

Comment:
Can you update the code so that `FastTree` is used when `n_threads=1`, and `FastTreeMP` is used when `n_threads` is any other value (see @gregcaporaso's [comment](https://github.com/qiime2/q2-phylogeny/issues/35#issuecomment-328945353))?